src/services/palindrome/PalindromeHandler.go

```diff
@@ -0,0 +1,76 @@
+package palindrome
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"src/models/palindrome"
+
+)
+
+
+type inputString struct {
+	Strings string
+}
+
+
+type outputStrings struct {
+	HasPalindrome []string `json:"has_palindrome"`
```

Comment:
output is a single string, not a slice of strings. 
and i dont think its correct to call this "has palindrome" because its not a bool

Reply:
output is actually a slice of strings.

Reply:
okay, now i realize it :D